dot-net-sdk/dto/BanditFlags.cs

```diff
@@ -2,11 +2,32 @@
 
 namespace eppo_sdk.dto;
 
-public interface IBanditFlags: IDictionary<string, BanditVariation[]> {
+public interface IBanditFlags : IDictionary<string, BanditVariation[]>
+{
     public bool IsBanditFlag(string FlagKey);
+
+    public bool TryGetBanditKey(string flagKey, string VariationValue, out string? BanditKey);
 }
 
 public class BanditFlags : Dictionary<string, BanditVariation[]>, IBanditFlags
 {
-    public bool IsBanditFlag(string flagKey) => this.Any(kvp=> kvp.Value.Any(bv => bv.FlagKey == flagKey));
+    public bool IsBanditFlag(string flagKey) => this.Any(kvp => kvp.Value.Any(bv => bv.FlagKey == flagKey));
+
+    public bool TryGetBanditKey(string flagKey, string variationValue, out string? banditKey)
+    {
+        banditKey = null;
```

Comment:
This is not returned? Unless the `out` keyword does something weird

Reply:
indeed `out` does something special: c# has return parameters. The `TryGet*` approach is idiomatic c# for checking if a key exists and getting the value in one call.